app/android/runtime_client/src/org/xwalk/app/runtime/extension/JsStubGenerator.java

```diff
@@ -0,0 +1,116 @@
+package org.xwalk.app.runtime.extension;
+
+import android.util.Log;
+import java.util.Map;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import org.xwalk.app.runtime.extension.ReflectionHelper.MemberInfo;
+
+public class JsStubGenerator {
+    static public String TAG = "JsStubGenerator";
+	ReflectionHelper reflection;
+	String jsHeader =
+        "var jsStub = requireNative(\"jsStub\").jsStub;\n" +
+        "var helper = jsStub.create(exports, extension);\n";
+	String jsEnder = "\n";
+	
+	JsStubGenerator (ReflectionHelper mirror) {
+		reflection = mirror;
+	}
+
+	String generate() {
+		String result = "";
+        if(reflection.getEventList() != null) {
+            result += generateEventTarget();
+        }
```

Comment:
indent, please go through your whole PR and fix them together. I will not point them all out to save review conversation log.